### PR TITLE
chore: improve error msgs for get-relying-party-info.sh

### PR DESCRIPTION
**Pull request by crabba**
Issue #, if available:

Description of changes:

Enhanced tests and comments in get-relying-party-info.sh:
* Added explicit tests for STAGE var defined by command line, environment variable, or defaulting to USER.
* Exit if config file not found.
* Added explicit tests for variables aws_profile, aws_region, solution_name.
* Exit if any of the above configuration values are not defined in config file.
* Expanded error messages when reading configuration file.
* Explicitly echo config file name and config values.

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Bingjiling** [comment]
Hi Andrew, 

Thanks for submitting this change! Could you please specify the tests you've done for these changes? 

Also, please change the base branch of the PR to develop branch and use [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/).

Please let me know if you have any questions.   

Thanks,
Yanyu

File: scripts/util.sh [hunk 1]
```diff
@@ -15,7 +15,20 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
...
+    echo "Setting STAGE from environment variable USER: ${USER}"
+    STAGE=$USER
+  else
+    echo "STAGE supplied as environment variable STAGE: ${STAGE}"
+  fi
+else
+  echo "STAGE supplied as command line argument: ${1}"
+  STAGE=$1
+fi
+# STAGE="${1:-${STAGE:-$USER}}"
```
Comment (carvantes) [inline scripts/util.sh]: remove the commented out code

File: scripts/util.sh [hunk 2]
```diff
@@ -29,7 +42,7 @@ popd > /dev/null
 
 function init_package_manager() {
   PACKAGE_MANAGER=pnpm
-  if ! command -v $PACKAGE_MANAGER; then
+    if ! command -v $PACKAGE_MANAGER &> /dev/null; then
     npm install -g pnpm@5.18.9
   fi
```
Comment (carvantes) [inline scripts/util.sh]: minor: `if` and `fi` should have the same identation level
```bash
if ! command -v $PACKAGE_MANAGER &> /dev/null; then
   npm install -g pnpm@5.18.9
fi
```

**Review by carvantes — COMMENTED** [review]
Thanks for submitting this PR. 

Looks like a nice improvement in usability for these scripts. I reviewed and the code is correct and should not cause any issues(gotta be careful with `util.sh` since it's used in many places)

I left only a couple very minor comments. Once they are addressed we can merge.

**Review by carvantes — APPROVED** [review]

**Pull request merged**